libs/editor/WordPressEditor/build.gradle

```diff
@@ -69,14 +69,21 @@ dependencies {
     implementation 'com.google.android.material:material:1.2.1'
     implementation 'org.apache.commons:commons-lang3:3.5'
     implementation 'com.android.volley:volley:1.1.1'
-    implementation 'org.wordpress:utils:1.25'
+    implementation 'org.wordpress:utils:trunk-1ed207c03d2242b6fc3d74f9e388e9163cbc82a6'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
     api ("$gradle.ext.aztecAndroidAztecPath:$aztecVersion") {
         exclude group: "com.android.volley"
+        exclude group: 'org.wordpress', module: 'utils'
     }
     api ("$gradle.ext.aztecAndroidWordPressShortcodesPath:$aztecVersion")
     api ("$gradle.ext.aztecAndroidWordPressCommentsPath:$aztecVersion")
 
+    implementation ("$gradle.ext.wputilsBinaryPath") {
+        version {
```

Comment:
Same as a few comments above: Uses `strictly` to workaround gradle wrong conflict resolution for pinned vs commithash versions.